.github/workflows/ci.yml

```diff
@@ -1,6 +1,13 @@
 name: ci
 
-on: [push]
+on:
+  # diff aware on every PR
+  pull_request: {}
+  # allow manual triggering
+  workflow_dispatch: {}
```

Comment:
We might not need this one if there's no use case for running tests manually

Reply:
yea wasn't sure if we'd need this but also didn't think it would hurt to be able to run tests manually?

Reply:
Sweet, yeah no issue with it b/c only folks with write access can trigger it